Cue/app/tabs/library/CardListView.js

```diff
@@ -13,6 +13,7 @@ import ListViewHairlineSeparator from '../../common/ListViewHairlineSeparator'
 type Props = {
   cards: Array<Card>,
   isFiltering: boolean,
+  accession: ?string,
```

Comment:
This actually needs to be `accession?: string` for Flow to type-check this correctly since the `accession` prop itself might be completely missing, rather than the prop occasionally having a `null` value.